### PR TITLE
Api schema

### PR DESCRIPTION
**Pull request by VictoriaBeilsten-Edmands**
Generate openapi.json in GH action and display schema in Storybook page using Swagger

### CODE REVIEW

File: .github/workflows/deploy-storybook.yaml [hunk 1]
```diff
@@ -41,4 +56,4 @@ jobs:
         checkout: false
         path: storybook/storybook-static
         install_command: echo Already done
-        build_command: pnpm run docs && pnpm build:storybook
+        build_command: pnpm run docs && pnpm build:storybook && cp openapi.json storybook/storybook-static/
```
Comment (VictoriaBeilsten-Edmands) [inline .github/workflows/deploy-storybook.yaml:59]: `cp openapi.json storybook/storybook-static/` is last as anything in `storybook/storybook-static` gets wiped in build step.

File: storybook/package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "@types/node": "^20.10.4",
     "@types/react": "^18.2.48",
     "@types/react-dom": "^18.2.18",
+    "@types/swagger-ui-react": "^4.18.3",
```
Comment (PeterC-DLS) [inline storybook/package.json:35]: Also need to add the actual package

File: .github/workflows/deploy-storybook.yaml [hunk 3]
```diff
@@ -28,6 +28,21 @@ jobs:
...
+
+    - name: Install server dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install ruff
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+    - name: Generate OpenAPI JSON
+      run: |
+        python server/davidia/generate_openapi.py
```
[the remark this answers is not in the document]
Comment (VictoriaBeilsten-Edmands) [reply .github/workflows/deploy-storybook.yaml:44]: To find the python or the script?

[the remark this answers is not in the document]
Comment (PeterC-DLS) [reply .github/workflows/deploy-storybook.yaml:43]: As the readme says!

**Review by PeterC-DLS — APPROVED** [review]

**Pull request merged**